content/en/profiler/enabling/go.md

```diff
@@ -95,6 +95,26 @@ Alternatively you can set profiler configuration using environment variables:
 | `DD_VERSION`                                     | String        | The [version][8] of your service. |
 | `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
 
+### Showing C function calls in CPU profiles
+
+Go's CPU profiler only shows detailed information for Go code by default. If your program calls into C code, time spent running C code will be reflected in the profile, but the call stacks will only show Go function calls.
+
+You can optionally use a library such as [ianlancetaylor/cgosymbolizer][10] to add detailed C function call information to CPU profiles. To use the library:
+
+1. Download the pacakge:
+
+    ```shell
+   go get github.com/ianlancetaylor/cgosymbolizer@latest
+    ```
+
+2. Add the following import anywhere in your program:
+
+    ```Go
+    import _ "github.com/ianlancetaylor/cgosymbolizer"
+    ```
+
+**Note**: This library is considered experimental. At Datadog, it has been used in production with few issues. However, the library can cause (infrequent) deadlocks in programs which use C++ exceptions, or which use libraries such as `tcmalloc` that also collect call stacks.
```

Comment:
```suggestion
**Note**: This library is considered experimental. It can cause (infrequent) deadlocks in programs that use C++ exceptions, or that use libraries such as `tcmalloc`, which also collect call stacks.
```

Reply:
I've committed this suggestion, but I was wondering if there's a particular reason why the sentence mentioning our usage at Datadog was removed? If there's a general policy around such things, it would be good to know so I can avoid issues in the future :)

Reply:
sorry @nsrip-dd I just saw this! there isn't a general policy around it, but in this case I didn't think its usage at Datadog was directly relevant to the note. I think you meant for this to explain that the library is relatively stable and that these issues aren't super common, but I just didn't think that level of nuance is really necessary here.

Reply:
No worries! That makes sense to me. Thank you for clarifying!